### ISSUE TITLE
gnocchiclient on pip is out of date or wrong

### ISSUE THREAD

**Issue by dhrp** [gnocchixyz/python-gnocchiclient#107]
I installed gnocchi with pip following the instructions here: https://gnocchi.xyz/gnocchiclient/installation.html

so:
https://pypi.org/project/gnocchiclient/#description

But it installs something different than shown here. Specifically, it fails immedately with the error:

```
    os.environ.set("OS_AUTH_TYPE", "password")
AttributeError: '_Environ' object has no attribute 'set'
```

but os.environ.set doesn't exist in python3, nor in python2. So it looks like someone just pushed a bogus version to pypi.

**Comment by jd** [comment]
This has been fixed in git, but never got released I think.

**Comment by dhrp** [comment]
This is what it looks like, though pip says it's at release 7.0.5, wheras github is at 7.0.4.

**Comment by jd** [comment]
Indeed, there's something wrong. Let me check that.

**Comment by jd** [comment]
No clue what we messed up with 7.0.5. I've released 7.0.6 manually since the CI is broken.
